wikimedia/wikipedia-ios
# Use BackgroundTasks.framework on iOS 13 and above

### Notes
Potential fix for issues with infrequent background updates 

### Test Steps
0. Ensure background updates are enabled for the app in system Settings
1. Use the app during the day
2. Don't use the app overnight
3. Check the app the next day to verify content has loaded
